spec/factories/state_file_nj_intakes.rb

```diff
@@ -103,6 +103,26 @@
       intake.raw_direct_file_data = intake.direct_file_data.to_s
     end
 
+
+    trait :mfs_complete_spouse do
```

Comment:
[Pebble] introduced something similar in my commit - can we merge the two traits?

https://github.com/codeforamerica/vita-min/blob/main/spec/factories/state_file_nj_intakes.rb#L115